Mathlib/CategoryTheory/Galois/EssSurj.lean

```diff
@@ -207,8 +210,12 @@ private def coconeQuotientDiagIsColimit :
   uniq s f hf := by
     apply Action.hom_ext
     ext (x : Aut F ⧸ V.toSubgroup)
-    induction x using Quotient.inductionOn
-    simp [← hf (SingleObj.star _)]
+    induction x using Quotient.inductionOn with | _ σ
+    dsimp
+    rw [← hf (SingleObj.star _)]
+    dsimp
+    simp only [← ConcreteCategory.comp_apply, Action.inv_hom_hom, Category.id_comp,
+      quotientToQuotientOfLE_hom_mk]
```

Comment:
This block can be replaced by: `simp [← hf (SingleObj.star _), ← ConcreteCategory.comp_apply]`

(The github comment interface seems a bit buggy and sometimes doesn't allow me to make code suggestions, I'm not sure what's going on, sorry)